docs/pt/docs/python-types.md

```diff
@@ -0,0 +1,314 @@
+# Introdução aos tipos Python
+
+**Python 3.6 +** tem suporte para "dicas de tipo" opcionais.
```

Comment:
I'm not sure about translating "type hint" into "dicas de tipo". In the main page (https://fastapi.tiangolo.com/pt/) we have used the English version instead. What do you think?

Reply:
I was in doubt, but I agree with you, I think keeping the standard in English in this case is better. I'll fix that.

Reply:
Done!! 🙂